h2o-admissibleml/src/main/java/hex/schemas/InfogramV3.java

```diff
@@ -0,0 +1,236 @@
+    public double total_information_threshold;
+
+    @API(help = "fraction of training frame to use to build the infogram model.  Default to 1.0",
+            level = API.Level.secondary, gridable = true)
+    public double data_fraction;
+
+    @API(help = "number of top k variables to consider based on the varimp.  Default to 0.0 which is to consider" +
+            " all predictors",
+            level = API.Level.secondary, gridable = true)
+    public int top_n_features;
+    
+    @API(help = "If true will calculate the p-value. Default to false",
+            level = API.Level.secondary, gridable = false)
+    public boolean compute_p_values;  // todo implement this option
+
+    public InfogramModel.InfogramParameters fillImpl(InfogramModel.InfogramParameters impl) {
+      super.fillImpl(impl);
+      if (infogram_algorithm_params != null && !infogram_algorithm_params.isEmpty()) {
+        Properties p = generateProperties(infogram_algorithm_params);
+        ParamNParamSchema schemaParams = generateParamsSchema(infogram_algorithm);
+        schemaParams._paramsSchema.init_meta();
+        impl._infogram_algorithm_parameters = (Model.Parameters) schemaParams._paramsSchema
+                .fillFromImpl(schemaParams._params)
+                .fillFromParms(p, true)
+                .createAndFillImpl();
+        super.fillImpl(impl);
+      }
+      return impl;
+    }
+    
+    Properties generateProperties(String algoParms) {
+      Properties p = new Properties();
+      HashMap<String, String[]> map = new Gson().fromJson(algoParms, new TypeToken<HashMap<String, String[]>>() {
+      }.getType());
+      for (Map.Entry<String, String[]> param : map.entrySet()) {
+        String[] paramVal = param.getValue();
+        if (paramVal.length == 1) {
+          p.setProperty(param.getKey(), paramVal[0]);
+        } else {
+          p.setProperty(param.getKey(), Arrays.toString(paramVal));
+        }
+      }
+      return p;
+    }
+    
+    private class ParamNParamSchema {
+      private ModelParametersSchemaV3 _paramsSchema;
+      private Model.Parameters _params;
+      
+      public ParamNParamSchema(ModelParametersSchemaV3 schema, Model.Parameters params) {
+        _paramsSchema = schema;
+        _params = params;
+      }
+    }
+
+    ParamNParamSchema generateParamsSchema(InfogramModel.InfogramParameters.Algorithm chosenAlgo) {
+      ModelParametersSchemaV3 paramsSchema;
+      Model.Parameters params;
+      switch (chosenAlgo) {
+        case AUTO:
+        case glm:
+          paramsSchema = new GLMV3.GLMParametersV3();
+          params = new GLMModel.GLMParameters();
+          // FIXME: This is here because there is no Family.AUTO. It enables us to know if the user specified family or not.
+          // FIXME: Family.AUTO will be implemented in https://0xdata.atlassian.net/projects/PUBDEV/issues/PUBDEV-7444
+          ((GLMModel.GLMParameters) params)._family = null;
```

Comment:
`Family.AUTO` already exists. I need to update the Stacked Ensemble code to use that...

Reply:
Tomas:  Not sure I understand what is going on here?  Can you explain please?

Reply:
I can explain how it was used in SE. In SE, we needed to know if a particular distribution/family was specified by a user or if we should try to guess (using the distributions/families from the base models).

Since at the time of writing there was no `AUTO` option I had to use `null` value to designate "unspecified family" to let the algo know that it should attempt to infer it.

Reply:
Got it.  I will add it.